docs/transfer-setup.md

```diff
@@ -0,0 +1,178 @@
+# Transfer your setup from a covenant signer to a covenant emulator
```

Comment:
This does not look to be a transition guide at all. All of the transition materials have been covered in the `covenant-signer` guide and this one assumes that there's a properly set up covenant-signer. We can work with this assumption and not refer to anything about transitioning.